src/apps/search/panels/BasePanel.tsx

```diff
@@ -24,6 +24,7 @@ import {
   useState
 } from 'react';
 import _ from 'underscore';
+import PanelHistoryContext from '../PanelHistoryContext';
```

Comment:
For consistency with other imports, let's use the absolute path instead of relative.